libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -37,7 +37,7 @@ const AP_Param::GroupInfo AP_MotorsHeli_Single::var_info[] = {
 
     // @Param: GYR_GAIN
     // @DisplayName: External Gyro Gain
-    // @Description: PWM in microseconds sent to external gyro on ch7 when tail type is Servo w/ ExtGyro
+    // @Description: PWM in microseconds sent to external gyro on the autopilot's servo/motor output7 when tail type is Servo w/ ExtGyro
```

Comment:
Please make "servo/motor output 7" just "motor 7 output" (suggest changes isn't working for me for some reason).  Trying to avoid confusion amongst users between the motor 7 output and the servo channel output.  I have tested in SITL and confirm that the out put moves with motor 7.  It is not enforced on servo channel 7.